packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.styles.ts

```diff
@@ -108,7 +108,7 @@ export const getStyles = (props: ITeachingBubbleStyleProps): ITeachingBubbleStyl
       {
         padding: 20
       },
-      isWide && {
+      !isWide && {
```

Comment:
I don't think we want to flip the boolean here since that max width styling does not currently exist on teaching bubbles that aren't wide. It looks like this was originally added here for teaching bubbles with images + text: https://github.com/OfficeDev/office-ui-fabric-react/pull/3441. Can we check to see that that scenario still works if we remove this whole section

```
isWide && {
  maxWidth: 302
}
```

Reply:
I looked at the PR deploy examples of images + text and they look the same as before:

![image](https://user-images.githubusercontent.com/4161791/65464029-cbfff080-de0d-11e9-9f08-6eda2f68eec4.png)
![image](https://user-images.githubusercontent.com/4161791/65464040-d15d3b00-de0d-11e9-9a91-2cefbd5964df.png)

Reply:
We probably don't need that styling at all, though, is what I'm thinking (since it's being applied on teachingbubbles that are not wide and that style wasn't there before)

Reply:
Ohhhh I see ok so I tried it with this section removed & the wide illustration one breaks without that isWide styling section:
![image](https://user-images.githubusercontent.com/4161791/65730734-b3930e80-e077-11e9-8fd9-5dd73353fe9a.png)

Reply:
Actually I tried this again after getting a clean build & updating to latest & removing this styling preserves the wide examples & the image+text examples:
![image](https://user-images.githubusercontent.com/4161791/66435237-40738b80-e9d9-11e9-99f6-e34a869d0be1.png)
![image](https://user-images.githubusercontent.com/4161791/66435244-44071280-e9d9-11e9-8143-4c416300500a.png
I'll push this change with the updated snaps